centrifuge/src/main/java/io/github/centrifugal/centrifuge/Client.java

```diff
@@ -1045,11 +1038,11 @@ private void cleanCommandFuture(Protocol.Command cmd) {
      * @param data:   a custom payload for RPC call.
      * @param cb:     will be called as soon as rpc response received or error happened.
      */
-    public void rpc(String method, byte[] data, ResultCallback<RPCResult> cb) {
+    public void rpc(@Nullable String method, byte[] data, ResultCallback<RPCResult> cb) {
```

Comment:
there's a check
```
if (method != null) {
    builder.setMethod(method);
}
```
so it's expected for this parameter to be null, right?

Reply:
Actually no, think it's an artefact from previous versions. Method can be an empty string – this is acceptable, but never null

Reply:
cool, updated